packages/wonder-blocks-toolbar/docs.md

```diff
@@ -0,0 +1,192 @@
+> TODO(scottgrant): Finish building, and write more thorough docs.
+
+## Content toolbar
```

Comment:
Grouping the examples into specific toolbar types isn't necessary.  It might've made sense if there was a specific `type` or `kind` prop but we don't have one.

Reply:
I wasn't sure whether the intent was to reproduce every example from our design or to just take the core set. This makes sense though, thanks!